### ISSUE TITLE
Check for no return type hint in docstring

### ISSUE THREAD

**Issue by mattibo** [jsh9/pydoclint#243]
Hi,

is there currently a way to configure pydoclint to check if the return type hint is not defined in the docstring?

I want to have only the type in the signature.

For arguments I use the option `arg-type-hints-in-docstring = false` but I`m missning a way to set this for the return type.

Thanks a lot!

**Comment by mattibo** [comment]
Hi @jsh9,

thank you for you answer. The option `--check-return-types` is turned on by default and not what i asked for.

A little example:

```python
def function_a(parameter_1: int) -> list[int]:
    """Summary.

    Args:
        parameter_1: description

    Returns:
        description
    """
    return [1]
```

This docstring should be valid for pydoclint and should not throw any errors.

With the given example following error is thrown:
```
1: DOC203: Function `function_a` return type(s) in docstring not consistent with the return annotation. Return annotation types: ['list[int]']; docstring return section types: ['']
```

**Comment by miraculixx** [comment]
I have a similar issue/example that gives DOC302 but is valid IMO.

```
def foobar(x):
    """ return the foobar tuple of x

    Args:
       x (int): the x

    Returns:
       the foobar tuple

       - x (int): x part
       - y (float): y part
    """
    return x, float(x)
```

`5: DOC203: Function `foobar` return type(s) in docstring not consistent with the return annotation. Return annotation has 0 type(s); docstring return section has 1 type(s).`

Running like this `pydoclint -aths FALSE -athd TRUE -ao TRUE --style google .`
